docs/take-streaming-backup.md

```diff
@@ -57,16 +57,16 @@ were both compressed and encrypted, they must be decrypted before they are uncom
 
 |Task  | Command  |
 |---------|------|
-| Stream the backup into an archived named `backup.xbstream` | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`|
-| Stream the backup into a compressed archive named `backup.xbstream`| `$ xtrabackup --backup --stream=xbstream --compress --target-dir=./ > backup.xbstream` |
-| Encrypt the backup | `$ xtrabackup --backup --stream=xbstream > backup.xbstream gzip -`` | openssl des3 -salt -k “password” backup.xbstream.gz.des3` |
-| Unpack the backup to the current directory | `$ xbstream -x <  backup.xbstream`
-| Send the backup compressed directly to another host and unpack it | `$ xtrabackup --backup --compress --stream=xbstream --target-dir=./ | ssh user@otherhost "xbstream -x"`|
-| Send the backup to another server using `netcat` | On the destination host:<br />`$ nc -l 9999 | cat - > /data/backups/backup.xbstream`<br /><br />On the source host:<br />`$ xtrabackup --backup --stream=xbstream | nc desthost 9999` |
-| Send the backup to another server using a one-liner  | `$ ssh user@desthost “( nc -l 9999 > /data/backups/backup.xbstream & )” && xtrabackup --backup --stream=xbstream | nc desthost 9999` |
-| Throttle the throughput to 10MB/sec using the [pipe viewer](https://www.ivarch.com/programs/quickref/pv.shtml) tool | `$ xtrabackup --backup --stream=xbstream | pv -q -L10m ssh user@desthost “cat - > /data/backups/backup.xbstream”` |
-| Checksum the backup during the streaming  | On the destination host:<br />`$ nc -l 9999 | tee >(sha1sum > destination_checksum) > /data/backups/backup.xbstream`<br /><br />On the source host:<br />`$ xtrabackup --backup --stream=xbstream | tee >(sha1sum > source_checksum) | nc desthost 9999`<br /><br />Compare the checksums on the source host:<br />`$ cat source_checksum 65e4f916a49c1f216e0887ce54cf59bf3934dbad`<br /><br />Compare the checksums on the destination host:<br />`$ cat destination_checksum 65e4f916a49c1f216e0887ce54cf59bf3934dbad` |
-| Parallel compression with parallel copying backup | `$ xtrabackup --backup --compress --compress-threads=8 --stream=xbstream --parallel=4 --target-dir=./ > backup.xbstream`|
+| Stream the backup into an archived named `backup.xbstream` | `xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`|
+| Stream the backup into a compressed archive named `backup.xbstream`| `xtrabackup --backup --stream=xbstream --compress --target-dir=./ > backup.xbstream` |
```

Comment:
why are we still seeing --target-dir=. ? 
Another comment is that marce made the xbstream optional parameter(this is only allowed value). So it can be just --stream(without =xbstream)?

Reply:
May be he has done it in 8.0 series too, can you please check? (@altmannmarcelo ) :)

Reply:
IIRC `--stream` does not require a parameter after FIFO work, which is in both 8.0 and Innovation ;)

Reply:
Sorry to draw your attention marce :D Just checking if you are still active on github ;) Thanks for the confirmation.

Reply:
Thank you @altmannmarcelo! :)